tools/payments/cmd/chain-account/README.md

```diff
@@ -0,0 +1,7 @@
+# Using GenerateSol
+
+```bash
+go run main.go -v \
```

Comment:
nit: can you update this to match the docs of the flags. looks like some of the flags have been renamed.